### ISSUE TITLE
"Tags" sections are anchored back to overview page

### ISSUE THREAD

**Issue by Xymph** [phpDocumentor/phpDocumentor#3042]
When a docblock contains a `@link` tag, a "Tags" section is generated with that `link` entry and its URL. The section header itself is anchored to `<project_path>/#tags` which is not defined on that project overview page.
(Hope this is not another existing issue/PR.)

## Expected behavior

Anchor to something meaningful, or don't anchor.

## Actual behavior

See intro. Example in `$maxlag` section in first screenshot of #3039. I'm not sure what the intended purpose of /#tags currently is.

## Steps to reproduce the problem

phpDocumentor.phar run -v --title=Wikimate -d . -i examples.php --sourcecode -t <path_to_my_private_site> -s graphs.enabled=true

## Your environment

* Version used: 3.1.2
* Install method: downloaded PHAR
* PHP version: 7.4
* Operating system and version: Ubuntu 20.04 LTS
* Link to your project: https://github.com/hamstar/Wikimate

**Comment by mvriel** [comment]
Last week, I thought I had fixed all these links; but just to be sure I checked your report. And apparently, I had still missed that. Thus I just issued PR https://github.com/phpDocumentor/phpDocumentor/pull/3404 to fix this and move the anchors onto a component so that others may more easily re-style them if needed.

Thank you for your report and this will be fixed as soon as the PR is approved and merged
